src/TesApi.Web/DeleteOrphanedBatchJobsHostedService.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Tes.Models;
+using Tes.Repository;
+
+namespace TesApi.Web
+{
+    /// <summary>
+    /// Background service to delete active Batch jobs that have no tasks as the result of job creation exception.
+    /// </summary>
+    public class DeleteOrphanedBatchJobsHostedService : BackgroundService
+    {
+        private static readonly TimeSpan runInterval = TimeSpan.FromHours(4);
```

Comment:
Should this check more frequently, to prevent potential hours of wasted VM time?  Maybe run every minute?  5 min?  20 min?

Reply:
Increased frequency to 1h. This happens very rarely, so don't want to burden Batch with too much continuous querying.